templates/etc/systemd/system/keepalived.service.j2

```diff
@@ -4,6 +4,8 @@
 Description=Keepalived
 After=network-online.target
 Wants=network-online.target
+# Only start if there is a configuration file
+ConditionFileNotEmpty={{ keepalived_configuration_file }}
```

Comment:
Why wouldn't there be a configurartion file?

Reply:
This is actually to avoid failed systemd units on which i alert in my monitoring system and i can do the installation of keepalived while baking my baseimage.